web/src/containers/ActivityDetailStandardsAndConditions.js

```diff
@@ -23,36 +23,21 @@ class ActivityDetailStandardsAndConditions extends Component {
     return (
       <Collapsible title={t('activities.standardsAndConditions.title')}>
         <p>{t('activities.standardsAndConditions.subheader')}</p>
-        {STANDARDS.map(std => {
-          const inputId = `a-${activity.id}-standards-${std.id}`;
```

Comment:
don't need this anymore (since we're leveraging the `Textarea` component which uses the name prop and automatically applies that to the corresponding label `htmlFor`)